client/my-sites/site-settings/jetpack-ads.jsx

```diff
@@ -138,6 +152,90 @@ class JetpackAds extends Component {
 							) }
 						</div>
 					</FormFieldset>
+
+					<hr />
+					<FormSectionHeading>{ translate( 'Privacy and Consent' ) }</FormSectionHeading>
+					<FormFieldset>
+						<SupportInfo
+							text={ translate(
+								'Enables a targeted advertising opt-out link for California consumers, as required by the California Consumer Privacy Act (CCPA).'
+							) }
+							link={
+								siteIsAtomic
+									? 'https://wordpress.com/support/your-wordpress-com-site-and-the-ccpa/'
+									: 'https://jetpack.com/support/ads/'
+							}
+						/>
+						{ this.renderToggle(
+							'wordads_ccpa_enabled',
+							! wordadsModuleActive,
+							translate( 'Enable targeted advertising to California site visitors (CCPA)' )
+						) }
+
+						<div className="site-settings__child-settings">
+							<FormSettingExplanation>
+								{ translate(
+									'For more information about the California Consumer Privacy Act (CCPA) and how it pertains to your site, please consult our {{a}}CCPA guide for site owners{{/a}}.',
+									{
+										components: {
+											a: (
+												<a
+													href={
+														siteIsAtomic
+															? 'https://wordpress.com/support/your-wordpress-com-site-and-the-ccpa/'
+															: 'https://jetpack.com/support/ads/'
+													}
+													target="_blank"
+													rel="noopener noreferrer"
+												/>
+											),
+										},
+									}
+								) }
+							</FormSettingExplanation>
+						</div>
+					</FormFieldset>
+
+					{ fields.wordads_ccpa_enabled && (
+						<div className="site-settings__child-settings">
+							<FormFieldset>
+								<FormLabel>{ translate( 'Do Not Sell Link' ) }</FormLabel>
+								<span>
+									{ translate(
+										'CCPA requires that you place a "Do Not Sell My Personal Information" link on every page of your site where targeted advertising will appear. You can use the {{a}}Do Not Sell Link (CCPA) Widget{{/a}}, or the {{code}}[ccpa-do-not-sell-link]{{/code}} shortcode to automatically place this link on your site. Note: the link will always display to logged in administrators regardless of geolocation.',
+										{
+											components: {
+												a: <a href={ widgetsUrl } target="_blank" rel="noopener noreferrer" />,
+												code: <code />,
+											},
+										}
+									) }
+								</span>
+								<FormSettingExplanation>
+									{ translate(
+										'Failure to add this link will result in non-compliance with CCPA.'
+									) }
+								</FormSettingExplanation>
+							</FormFieldset>
+							<FormFieldset>
+								<FormLabel htmlFor="ccpa-privacy-policy-url">
+									{ translate( 'Privacy Policy URL' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **111** times:
`translate( 'Privacy Policy' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wordads-ccpa)